code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -0,0 +1,400 @@
+		C.adjustBruteLoss(-5)
+		C.adjustFireLoss(-5)
+		C.adjustOxyLoss(-5)
+		C.adjustToxLoss(-5)
+
+/mob/living/simple_animal/hostile/guardian/ranged
+	a_intent = "help"
+	melee_damage_lower = 10
+	melee_damage_upper = 10
+	damage_transfer = 1.2
+	projectiletype = /obj/item/projectile/neurotox
+	projectilesound = 'sound/weapons/pierce.ogg'
+	ranged = 1
+	rapid = 1
+	range = 13
+	playstyle_string = "As a ranged type, you have only light damage resistance, but are capable of spraying neurotoxin."
+
+/mob/living/simple_animal/hostile/guardian/bluespace
+	ranged = 1
+	range = 15
+	melee_damage_lower = 10
+	melee_damage_upper = 10
+	speed = -1
+	projectiletype = /obj/item/projectile/magic/teleport
+	projectilesound = 'sound/weapons/emitter.ogg'
+	playstyle_string = "As a bluespace type, you have only light damage resistance, but are capable of shooting teleporation bolts as well as flinging enemies away with your standard attack."
+
+/mob/living/simple_animal/hostile/guardian/bluespace/AttackingTarget()
+	..()
+	if(target != anchored && target != src.summoner)
+		do_teleport(target, target, 10)
+
+
+
+
+
+////////Creation
+
+/obj/item/weapon/guardiancreator
+	name = "deck of tarot cards"
+	desc = "An enchanted deck of tarot cards, rumored to be a source of unimaginable power. "
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "deck_syndicate_full"
+	var/used = FALSE
+	var/theme = "magic"
+	var/mob_name = "Guardian Spirit"
+	var/use_message = "You shuffle the deck..."
+	var/used_message = "All the cards seem to be blank now."
+	var/failure_message = "..And draw a card! It's...blank? Maybe you should try again later."
+
+/obj/item/weapon/guardiancreator/attack_self(mob/living/user)
+	if(used == TRUE)
+		user << "[used_message]"
+		return
+	used = TRUE
+	user << "[use_message]"
+	var/list/candidates = get_candidates(BE_ALIEN, ALIEN_AFK_BRACKET)
+
+	shuffle(candidates)
+
+	var/time_passed = world.time
+	var/list/consenting_candidates = list()
+
+	for(var/candidate in candidates)
+
+		spawn(0)
+			switch(alert(candidate, "Would you like to play as the [mob_name] of [user.real_name]? Please choose quickly!","Confirmation","Yes","No"))
+				if("Yes")
+					if((world.time-time_passed)>=50 || !src)
+						return
+					consenting_candidates += candidate
+
+	sleep(50)
+
+	if(!src)
+		return
+
+	if(consenting_candidates.len)
+		var/client/C = null
+		C = pick(consenting_candidates)
+		spawn_guardian(user, C.key)
+	else
+		user << "[failure_message]"
+		used = FALSE
+
+
+/obj/item/weapon/guardiancreator/proc/spawn_guardian(var/mob/living/user, var/key)
+	var/gaurdiantype = pick("Fire", "Standard", "Scout", "Shield", "Ranged", "Healer", "Fast")
+	var/pickedtype = /mob/living/simple_animal/hostile/guardian/punch
+	var/picked_color = randomColor(0)
+	switch(gaurdiantype)
+
+		if("Fire")
+			pickedtype = /mob/living/simple_animal/hostile/guardian/fire
+			switch (theme)
+				if("magic")
+					user << "..And draw Atmosia, bringer of cleansing fires!"
```

Comment:
can you please put these strings on the mobs?
eg:
var/magic_fluff_string
var/tech_fluff_string
var/bio_fluff_string

and then you can do ```user << initial(pickedype.magic_fluff_string)``` (you'll need to replace var/pickedtype with ```var/mob/living/simple_animal/hostile/guardian/pickedtype```